app/views/admin/timeline/_step.html.slim

```diff
@@ -29,15 +29,15 @@
     class="#{classes} step-date step-start-date"
     data-id=entry.id
     data-precedent-id=entry.schedule_parent_id]
-    = entry.display_start_date ? ldate(entry.display_start_date, format: :short) : ""
+    = ldate(entry.display_start_date, format: :short) if entry.display_start_date
     - if entry.schedule_parent_id
       i.fa.fa-link
   td[
     class="#{classes} step-date step-end-date"
     data-id=entry.id
     data-precedent-id=entry.schedule_parent_id
     data-dependent-step-start-date=ldate(entry.display_end_date)]
-    = entry.display_end_date ? ldate(entry.display_end_date, format: :short) : ""
+    = ldate(entry.display_end_date, format: :short) if entry.display_end_date
```

Comment:
The above `if`s are not necessary -- that's what `ldate` does. If the value is nil, it returns empty string.

When you're refactoring, it's good to think through the logic of the preexisting code. Don't assume it's correct or necessary! You might be able to clean it up even more :)